packages/xo-web/src/common/intl/messages.js

```diff
@@ -98,10 +98,10 @@ const messages = {
   newVmBackup: 'New VM backup',
   editVmBackup: 'Edit VM backup',
   backup: 'Backup',
-  rollingSnapshot: 'Rolling Snapshot',
-  deltaBackup: 'Delta Backup',
-  disasterRecovery: 'Disaster Recovery',
-  continuousReplication: 'Continuous Replication',
+  rollingSnapshot: 'Rolling\u00A0Snapshot',
+  deltaBackup: 'Delta\u00A0Backup',
+  disasterRecovery: 'Disaster\u00A0Recovery',
+  continuousReplication: 'Continuous\u00A0Replication',
```

Comment:
Why don't you use the CSS `white-space` property instead?

Reply:
Do you mean &nbsp; ?

Reply:
My bad, how to use it in the message ?

Reply:
My bad, i see what you mean.